contributing/TESTING.md

```diff
@@ -410,7 +410,7 @@ You can create flamecharts for any node process used by the build system by leve
 Here's an example for `gulp dist --closure_concurrency=1`:
 
 ```sh
-yarn 0x -o node_modules/.bin/gulp dist --closure_concurrency=1
+npx 0x -o node_modules/.bin/gulp dist --closure_concurrency=1
 ```
 
-Important to node is `0x` will automatically create a flamechart and a serving folder locally within the repository, please don't add them to PRs!
+Important to note is `0x` will automatically create a flamechart and a serving folder locally within the repository, please don't add them to PRs!
```

Comment:
Not really part of this PR, but if the flamechart gets its own directory in the repo, we should probably add that to the gitignore

Reply:
Nice idea. Tested this out. The flamechart output directory has a variable name and is generated in whichever directory the command is called. Can't really `.gitignore` it.

Reply:
I believe the directory is always `amphtml/\d{7}\.0x`, so `*.0x/` would do the trick

Reply:
Interesting idea. The [docs](https://github.com/davidmarkclements/0x#generating) say that the filename string is a PID, which can vary in shape and size across OSs. Given the already large scope of this PR, and the fact that flamechart creation is not part of the routine development workflow, I'm going to leave this as is for now.